Bump package version to 2.1.0

roribio opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
Bump `package.json` version to `2.1.0` to create a release for the latest changes.

<!--- Describe your changes in detail -->

## Related Issue

<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

slitviachenko reviewed (changes requested):
Some questions have been brought up in https://adobedx.slack.com/archives/C039BPS6RUK/p1663014283496759?thread_ts=1663014145.980759&cid=C039BPS6RUK

roribio commented:
```shell
➜  aio-cli-plugin-app-templates git:(main) ✗ aio plugins
@adobe/aio-cli-plugin-app 8.6.1
@adobe/aio-cli-plugin-app-templates 1.1.0 (link) /Users/roribio/Source/aio-cli-plugin-app-templates
@magento/aio-cli-plugin-commerce-admin 0.0.3
➜  demo-app-builder-project git:(master) aio app use
You are currently in:
1. Org: DevX Acceleration Prod
2. Project: RonDemo
3. Workspace: Stage

? Switch to a new Adobe Developer Console configuration: B. Switch to another Workspace in the current Project
? Select a Workspace, or press + to create new: Stage
? The file /Users/roribio/Source/demo-app-builder-project/.env already exists: Overwrite
? The file /Users/roribio/Source/demo-app-builder-project/.aio already exists: Overwrite

✔ Successfully imported configuration for:
1. Org: DevX Acceleration Prod
2. Project: RonDemo
3. Workspace: Stage.
➜  demo-app-builder-project git:(master) aio templates discover
 Name                               Version   Description                                                Publish Date    Adobe Recommended 
 ────────────────────────────────── ───────── ────────────────────────────────────────────────────────── ─────────────── ───────────────── 
 @adobe/generator-app-asset-compute 0.2.3     Extensibility template for generator-aio-app               June 8, 2022    yes               
 @adobe/generator-app-excshell      0.2.2     Extensibility template for generator-aio-app               August 25, 2022 yes               
 @adobe/generator-app-api-mesh      0.1.0     Extensibility template for Adobe API Mesh, for App Builder August 30, 2022 yes               
➜  demo-app-builder-project git:(master) aio templates discover -i
? Select templates to install @adobe/generator-app-asset-compute@0.2.3
ℹ Installing template @adobe/generator-app-asset-compute
ℹ Installing npm package @adobe/generator-app-asset-compute

added 2 packages, removed 530 packages, changed 12 packages, and audited 414 packages in 11s

32 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
✔ Installed npm package @adobe/generator-app-asset-compute
ℹ Running template @adobe/generator-app-asset-compute
    force package.json
   create src/dx-asset-compute-worker-1/ext.config.yaml
    force app.config.yaml
    force .env
   create src/dx-asset-compute-worker-1/actions/worker/index.js
   create src/dx-asset-compute-worker-1/test/asset-compute/worker/corrupt-input/file.jpg
   create src/dx-asset-compute-worker-1/test/asset-compute/worker/corrupt-input/params.json
   create src/dx-asset-compute-worker-1/test/asset-compute/worker/simple-test/file.jpg
   create src/dx-asset-compute-worker-1/test/asset-compute/worker/simple-test/params.json
   create src/dx-asset-compute-worker-1/test/asset-compute/worker/simple-test/rendition.jpg

Changes to package.json were detected.

Running npm install for you to install the required dependencies.
added 523 packages, changed 1 package, and audited 937 packages in 35s

75 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
✔ Finished running template @adobe/generator-app-asset-compute
✔ Installed template @adobe/generator-app-asset-compute
```

roribio commented:
Repeated tests with changes from #11.

```shell
➜  demo-app-builder-project git:(master) ✗ aio templates discover -i
? Select templates to install @adobe/generator-app-asset-compute@0.2.3
ℹ Installing template @adobe/generator-app-asset-compute
ℹ Installing npm package @adobe/generator-app-asset-compute

added 49 packages, and audited 937 packages in 4s

75 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
✔ Installed npm package @adobe/generator-app-asset-compute
ℹ Running template @adobe/generator-app-asset-compute
    force package.json
    force src/dx-asset-compute-worker-1/ext.config.yaml
    force app.config.yaml
    force .env
   create src/dx-asset-compute-worker-1/actions/worker-1/index.js
   create src/dx-asset-compute-worker-1/test/asset-compute/worker-1/corrupt-input/file.jpg
   create src/dx-asset-compute-worker-1/test/asset-compute/worker-1/corrupt-input/params.json
   create src/dx-asset-compute-worker-1/test/asset-compute/worker-1/simple-test/file.jpg
   create src/dx-asset-compute-worker-1/test/asset-compute/worker-1/simple-test/params.json
   create src/dx-asset-compute-worker-1/test/asset-compute/worker-1/simple-test/rendition.jpg

Changes to package.json were detected.

Running npm install for you to install the required dependencies.

up to date, audited 937 packages in 2s

75 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
✔ Finished running template @adobe/generator-app-asset-compute
✔ Installed template @adobe/generator-app-asset-compute
```
